src/tests/unit/tests/scanner/__snapshots__/get-rule-inclusions.test.ts.snap

```diff
@@ -166,8 +170,7 @@ Object {
     "status": "included",
   },
   "identical-links-same-purpose": Object {
-    "reason": "rule is tagged best-practice",
-    "status": "excluded",
+    "status": "included",
```

Comment:
Do we expect [identical-links-same-purpose](https://github.com/dequelabs/axe-core/blob/f583c70b191513f5bd116a3ed091e937ea2cd65d/lib/rules/identical-links-same-purpose.json) to be included? The `best-practices` tag was removed in 4.4.0, it has the `wcag2aaa` tag, and in the tests we [declare it's omitted due to AAA](https://github.com/microsoft/accessibility-insights-web/blob/5deba6151546207ecb802540184a0c7e46947827/src/tests/unit/tests/scanner/map-axe-tags-to-guidance-links.test.ts#L17).

Reply:
The team decided that the product will continue to exclude this rule.